### PR TITLE
update broadcaster to latest version with 0.3 futures dep

### PR DESCRIPTION
**Pull request by serzhiio**

### CODE REVIEW

**Comment by yoshuawuyts** [comment]
#565 was filed at the same time as this PR. If we can't get that patch to land before next release, we should merge this one. thanks for submitting this @serzhiio!

**Comment by leo60228** [comment]
I've released 1.0.0. It doesn't have any breaking changes with 0.3.0, though.

File: Cargo.toml [hunk 1]
```diff
@@ -53,7 +53,7 @@ std = [
 async-attributes = { version = "1.1.1", optional = true }
 async-macros = { version = "2.0.0", optional = true }
 async-task = { version = "1.0.0", optional = true }
-broadcaster = { version = "0.2.6", optional = true, default-features = false, features = ["default-channels"] }
+broadcaster = { version = "0.3.0", optional = true, default-features = false, features = ["default-channels"] }
```
Comment (stjepang) [inline Cargo.toml]:
```suggestion
broadcaster = { version = "1.0.0", optional = true, default-features = false, features = ["default-channels"] }
```

**Comment by k-nasa** [comment]
Close with merge of #681

**Pull request closed without merging**